RHEL6_7/networking/dovecot/check

```diff
@@ -69,13 +78,48 @@ file manually on the new system.
 PREF=$POSTUPGRADE_DIR/dovecot
 mkdir -p $PREF
 sed '2,/^#!\//d' $0 >$PREF/dovecot_postupgrade.sh
-if ! [[ -z "$uid_val" ]] && [ "$uid_val" -lt 1000 ];then
-  export uid_val ; perl -pi -e '$_ .= qq(sed -i -e "s/first_valid_uid = 1000/first_valid_uid = $ENV{"uid_val"}/g" /etc/dovecot/conf.d/10-mail.conf\n) if  /restorecon -R $CONFIG_FILE/' "$PREF"/dovecot_postupgrade.sh
-fi
 chmod +x $PREF/dovecot_postupgrade.sh
 
 rm -f $TMPF1 $TMPF2
-exit $RESULT_FIXED
+cd /etc/dovecot/
+
+#Get the the config, that actually provides the first_valid_uid directive
```

Comment:
Actually this is not true, there can be multiple includes and the value can be set even inside main configuration file, when includes are used. And even it can be included between includes mupltiple times - ok it doesn't make sense so much, but user can do that. Solution discussed in person.